third_party/txt/src/txt/paragraph.cc

```diff
@@ -423,6 +423,58 @@ bool Paragraph::ComputeBidiRuns(std::vector<BidiRun>* result) {
   return true;
 }
 
+void Paragraph::ComputeStrut(StrutMetrics* strut, SkFont& font) {
+  strut->ascent = 0;  // Positive value to keep signs clear.
+  strut->descent = 0;
+  strut->leading = 0;
+  strut->half_leading = 0;
+  strut->line_height = 0;
+  strut->force_strut = false;
+
+  // Minimally, a font size and either a line_height or leading must be defined
```

Comment:
Is this still accurate?  It looks like there is a valid default font size, and line_height and leading are no longer required for a valid strut